Source/Scene/Model.js

```diff
@@ -2588,7 +2819,20 @@ define([
                 var positionAttribute = primitive.attributes.POSITION;
                 if (defined(positionAttribute)) {
                     var a = accessors[positionAttribute];
-                    boundingSphere = BoundingSphere.fromCornerPoints(Cartesian3.fromArray(a.min), Cartesian3.fromArray(a.max));
+                    var extensions = a.extensions;
```

Comment:
This is the same logic as above, right?  Can we move it to a separate private function?